cartoframes/core/managers/context_manager.py

```diff
@@ -172,15 +172,15 @@ def _create_table_from_query(self, table_name, query, schema, cartodbfy=True):
             create=_create_table_from_query_query(table_name, query),
             cartodbfy=_cartodbfy_query(table_name, schema) if cartodbfy else ''
         )
-        self.execute_long_running_query(query)
+        self.execute_query(query)
```

Comment:
mm I guess we shouldn't change this. The SQL API enforces hard database timeouts, that was the reason we used the batch API for these cases. I guess we should stick to the previous method.

Reply:
I wasn't sure about this. The point is that I made a manual test and it worked with a simple SQL query and not with the Batch SQL. In any case, I agree to stick with the previous method and maybe discuss this change in the future.

Reply:
Yep, it may work, but it depends on the query to execute. This was one of the initial pains of the first implementation of CF.

Reply:
OK. Changed